omnigibson/utils/usd_utils.py

```diff
@@ -743,19 +744,44 @@ def _refresh(cls):
             cls.mark_valid()
 
     @classmethod
-    def get_world_pose(cls, prim_path):
+    def get_position_orientation(cls, prim_path, frame: Literal["world", "parent"] = "world"):
```

Comment:
removed the get_position_orientation with parent frame in poseAPI. however, this would cause the robot setter/getter to fail. we can discuss more on Friday.